<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridNodeAttributePropertyDialog</class>
 <widget class="QDialog" name="GridNodeAttributePropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>127</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="ScalarsToColorsEditWidgetContainer" name="scalarsToColorsWidget" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="contourSettingGroupBox">
     <property name="title">
      <string>Contour setting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="ContourSettingWidget" name="contourSettingWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="TransparencyWidget" name="transparencyWidget" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TransparencyWidget</class>
   <extends>QWidget</extends>
   <header>guibase/widget/transparencywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ContourSettingWidget</class>
   <extends>QWidget</extends>
   <header>guibase/widget/contoursettingwidget.h</header>
   <container>1</container>
   <slots>
    <signal>contourChanged()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>ScalarsToColorsEditWidgetContainer</class>
   <extends>QWidget</extends>
   <header>scalarstocolors/scalarstocolorseditwidgetcontainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GridNodeAttributePropertyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GridNodeAttributePropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contourSettingWidget</sender>
   <signal>contourChanged()</signal>
   <receiver>GridNodeAttributePropertyDialog</receiver>
   <slot>handleContourChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>handleContourChange()</slot>
 </slots>
</ui>
